testng-core/src/test/kotlin/test/SimpleBaseTest.kt

```diff
@@ -0,0 +1,489 @@
+package test
+
+import org.assertj.core.api.Assertions
+import org.testng.*
+import org.testng.annotations.ITestAnnotation
+import org.testng.collections.Lists
+import org.testng.internal.annotations.AnnotationHelper
+import org.testng.internal.annotations.DefaultAnnotationTransformer
+import org.testng.internal.annotations.IAnnotationFinder
+import org.testng.internal.annotations.JDK15AnnotationFinder
+import org.testng.xml.*
+import org.testng.xml.internal.Parser
+import java.io.*
+import java.nio.file.FileVisitResult
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.attribute.BasicFileAttributes
+import java.util.*
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.regex.Pattern
+import java.util.stream.Collectors
+import kotlin.collections.ArrayList
+
+open class SimpleBaseTest {
+
+    companion object Companion {
+        @JvmStatic
+        val TEST_RESOURCES_DIR = "test.resources.dir"
+
+        @JvmStatic
+        fun run(vararg testClasses: Class<*>) : InvokedMethodNameListener {
```

Comment:
```kotlin
@JvmStatic
fun run(vararg testClasses: Class<*>) = run(false, *testClasses)
```

Reply:
the function can be removed because you can set a default value on skipConfiguration

Reply:
This function is being used by child classes and I am trying to keep the changeset to as minimal as possible.